client/js/views/templates/AboutUs.js

```diff
@@ -1,3 +1,24 @@
 module.exports = function( { model } ) {
-    return `<div>about us</div>`
+	const images = model.reduce((memo, image) => {
+	  	const markup = `<div class='${ image.class }'>
+	    	<img data-src='${ this.ImageSrc(image.filename) }' alt='${ image.alt }'/> 
+	    	<div>${ image.date }<br/>${ image.caption }</div>
+	  	</div>`
+	  	return memo + markup
+	}, '')
+    return `<div>
+    	<div>
+	    	<div><img data-src='${ this.ImageSrc('FarmScene.jpg') }' alt='Farm Scene'/></div>
+	    	<div>
+	    		<div>Headline for About Us and the Farm</div><br/><br/>
```

Comment:
Use an <h*> tag for anything that's a heading. This one would probably be <h2>, maybe <h3>. Also, <br> tags should be used very sparingly. I can't remember the last time I used one. A div is by default `display: block` so it will be its own line. And instead of using two <br> tags to create extra distance, just set `margin-bottom` on the element according to how much space you need.